### PR TITLE
Update runtests and CI.yml for running stan test on CI

### PR DESCRIPTION
**Pull request by Vaibhavdixit02**
Attempt to migrate Stan tests to GitHub Actions

### CODE REVIEW

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -37,8 +37,7 @@ jobs:
       - uses: julia-actions/julia-runtest@v1
         env:
           GROUP: ${{ matrix.group }}
-          if: ${{ matrix.group == 'Stan' }}
-            JULIA_CMDSTAN_HOME: ${{ '$HOME/cmdstan-2.20.0/' }}
+          JULIA_CMDSTAN_HOME: ${{ '$HOME/cmdstan-2.20.0/' }}
```
Comment (ChrisRackauckas) [inline .github/workflows/CI.yml:40]:
```suggestion
          JULIA_CMDSTAN_HOME: ${{ '$HOME/cmdstan-2.26.1/' }}
```

**Comment by Vaibhavdixit02** [comment]
This doesn't make sense, the Turing tests seem to get stalled here on the CI whereas run fine locally 🤦

**Comment by ChrisRackauckas** [comment]
It might be due to code coverage calculations? Try turning them off.

**Comment by Vaibhavdixit02** [comment]
Well tests passed now and https://github.com/SciML/DiffEqBayes.jl/pull/211/checks?check_run_id=2425350425#step:8:1090 it looks like it is the tests that took so long?

**Pull request merged**